financialaid/serializers.py

```diff
@@ -0,0 +1,62 @@
+"""
+Serializers from financialaid
+"""
+import datetime
+from django.shortcuts import get_object_or_404
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from rest_framework.fields import IntegerField, FloatField, CharField
+
+from courses.models import Program
+from dashboard.models import ProgramEnrollment
+from financialaid.api import determine_tier_program, determine_auto_approval
+from financialaid.models import FinancialAid, FinancialAidStatus
+
+
+class FinancialAidSerializer(serializers.Serializer):
```

Comment:
Program is not on FinancialAid - it's connected via TierProgram.  We did spent a while trying to wrangle ModelSerializer to work, but decided it wasn't worth spending hours over when this current solution took us 5 minutes to make work.  I definitely agree with the interest of speeding this up.